grakn-test/src/test/java/ai/grakn/test/graql/reasoner/ReasoningTests.java

```diff
@@ -46,7 +48,7 @@ public void testSet10() {
         QueryBuilder iqb = testSet10.graph().graql().infer(true);
```

Comment:
This test is still named poorly and I am still unsure of what it is testing. It is testing graql file `testSet10.gql` but what about it exactly? 

As an outsider to reasoner if this test breaks I have no idea where to look to start fixing it.